### PR TITLE
[docs] Add a note that concurrency testing is not supported in Edge

### LINKED ISSUE #1664: Cann't start tests in concurrency mode in MSEdge
### Are you requesting a feature or reporting a bug?

Bug

### What is the current behavior?

One instance of Edge opens and loading page is shown infinite

### What is the expected behavior?

Test run in three instance of Edge

### How would you reproduce the current behavior (if this is a bug)?
```
testcafe -c 3 edge test.js
```

#### Provide the test code and the tested page URL (if applicable)

Any test

### Specify your

* operating system: Window 10
* testcafe version: 0.17.0-alpha6
* node.js version: 15.15063

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs 

fixes #1664

### CODE REVIEW

File: docs/articles/documentation/using-testcafe/common-concepts/concurrent-test-execution.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ These instances constitute the pool of browsers against which tests run concurre
 To enable concurrency, use the [-c or --concurrency](../command-line-interface.md#-c-n---concurrency-n)
 command line option or the [runner.concurrency](../programming-interface/runner.md#concurrency) method of the programming interface.
 
+> Important! Concurrent test execution is not supported in Microsoft Edge.
```
Comment (AlexanderMoskovkin) [inline docs/articles/documentation/using-testcafe/common-concepts/concurrent-test-execution.md]: Maybe we should describe why?

Comment (VasilyStrelyaev) [reply docs/articles/documentation/using-testcafe/common-concepts/concurrent-test-execution.md]: If we do, and when MS allows invoking multiple Edge windows, this note will immediately become false.

If we don't, it will still be true until we change `browser-natives` to support MS Edge.

That's why I don't like documenting the functionality of someone else's products.

**Comment by VasilyStrelyaev** [comment]
Or... I'd better add the reason. Because if this reason becomes outdated, someone will come across this note and tell us that we can implement support for MS Edge now.

File: docs/articles/documentation/using-testcafe/common-concepts/concurrent-test-execution.md [hunk 2]
```diff
@@ -21,6 +21,8 @@ These instances constitute the pool of browsers against which tests run concurre
 To enable concurrency, use the [-c or --concurrency](../command-line-interface.md#-c-n---concurrency-n)
 command line option or the [runner.concurrency](../programming-interface/runner.md#concurrency) method of the programming interface.
 
+> Important! Concurrent test execution is not supported in Microsoft Edge. This is because there is no known way to start Edge in a new window and make it open a particular URL.
```
Comment (kirovboris) [inline docs/articles/documentation/using-testcafe/common-concepts/concurrent-test-execution.md]: Did you check this [util](https://github.com/MicrosoftEdge/edge-launcher) out?

Comment (AlexanderMoskovkin) [reply docs/articles/documentation/using-testcafe/common-concepts/concurrent-test-execution.md]: /cc @AndreyBelym

Comment (AndreyBelym) [reply docs/articles/documentation/using-testcafe/common-concepts/concurrent-test-execution.md]: I have tried it, it's useless. It can't open URL in a new window. It can be used to create a "private browsing" window once, but the next URLs still will be opened in new tabs of this "private" window.

Comment (kirovboris) [reply docs/articles/documentation/using-testcafe/common-concepts/concurrent-test-execution.md]: Ok
